Documentation/Diagrams/UML-Model/ivms-model-uml-source.drawio

```diff
@@ -1 +1 @@
mt+bFe2Jy1bO0M47JuJN+3VcyjN1gzB/hbd9nS77PysUAXZ+DGbgx6XF7wIsKjF096jtIjNHBJtq39vFFBx0cvADCqzo66JTpPu2GNnyMmyUo85p50GSo0Bm1vz3++hTe/fjnE/MGl6H/8u/9JwwZYkQQT7PRggTtceh6lFfYbhKU1Fjl2n5/mE7XgOpWuvpJLxvo4cp12+qCoECTLgC5vGTt7GnIq6r5DSevYhj8hnHCL7WECS6Q2Dc5g12uq33j34evq6rTx+F0JQs8Zt3W+RRjU/D6anJiiVxfsgSEk9CXUTV3B0xf8rVX+iHxaNx3Q29te2H/BcQBXXX+VkfWf207j0kwn5LQ3Z6sW3Afl6cAobt4P3dxZ88pHLoO5azpFBvjx8ATgYK/eE3kbU5XaxZWqjseN2y3OBj7z9wNYhovEHpQ6Ku6aOFW4NOKAwVh296RkDIUPiz63QZH4CXoy9LDkqHBYGMg4Gd10hKbxCYQ8+J/5iy5wByNNE0waV2UfPbhy8UX/pnzsCceO5rPRGxPi+h05otAn7cej1wlt9/+Sl48eC38t2EILLvzUkIIzN15zKJk/KC/kZ9psMiqZwBg69sDgHaxD9A7MipATeHUtfJUA6QCJBW6h0cFq1DDxBuTrEHk7eaEjVng+pfrUt4az/l40Uvre33NDVvWlqjjXySOF+nwXVT/NmTkhcbfN97/ELfi7XVydJGN2ZcHi+wg8M7FBtXrLoeXXFHxvOl5XiHfNw+Suxp2dry+7/Iou3EprhGbh0Oyo/aylF9uqoxJlf5CVO1OnoTE5xbaE9n6HTvSRe4E1zf4ZW7za2VsZLdIfmn6qRx3akj/0DX7COnkUXfKAu9hQoPKrIIjVRbieZVU2aQiaFJZ2jap9HxeUQmpePW6i43L0na59HvM9vb32JqzebtXr9fTrL81p5NfUDPDi8Zzhuw7O6nXnCirlJZlLkrPHf4eL5XTX3XNvEMW/6T0261XiVNsyIJAJKMNsofSdnZtH7SzrpU1iCki2boo7yRg/qbWmb19CzYaRQSoQSvG3PWzs9MalVh2TkiSMLfZ7KhENiN5L3fmrmwj9GjW49Gs7NJswxm3RbdW340mZTpFp8ZO1R2PO1OX7um5v+VaMINTU/bW7bn85YN4Bo/brxqLJ0ujdpa0L/wdf6AJE1nVMiO2xN49pV6k4+RaBYlxuyprphvJNvo8KmvkYIzbqnbIqt2ANkScvHVr5caByTMVDJHCjTrd3IAnT0FoM1k/Iq/LfvZvRfN6f3oaVX0v7drN5Hc2SkdqQJZ1CG+2FE2je9axHd0ykv+7W1q0re6Zs3HafGsrkbcWOw1ai5IM7ZO3Fi1TtbUonXPQwBhwRIkvXr3VHMu/d/Rn5bpU0zaKoz8diBb/Cn5+GP/6+WX4FP28+/HN7nf73mrRur+ijx34TDgQcr3r0uOQG1cuy97aA0srWC92wPLrah8evosX5WHwN02zvlzmzFRYFR+9THu2KDkvk1N1/1Krhrw5eYvSOZ4WpYaWATT0JZemraxJKbE48xxsgxmKOyuknsaqwrr+2FjV1FhVXQ0MrLGSThbdmzpIGfAoii3zl0oTw7PAf/2cKVsIGFfv2HNa/25pHmScRf6TcfFQQHgbXL1D+otLd+/uceR41V4HI5Y2ABsluD0QHCOaXMdDTglcTBQQ3iaX/JfDa5Qovi9GUryHXyt+owQVD0mJBueQyt0bxTDUwbo3wIKSelUHxs7NNF93YLQOySVqFtt2RP5tyGcRr9eRr30qwPugL8bOaopGH0pkyjQr2NmNBqaKazidWJXnQ8RS132jVV70o55YlRdY3lXNcrA0l0Op8gLLZT7fRqtc5qTZw+VbzcZHv+9+tNENbV/HbxvKHCzdPAYdv3VahSuBHo/jFzeVgYRXteO3dE+ZPpvO3GCBe7tCM0C5oxc3lYGEV7mjt3RTmf/RWZ95qG5g+FU7de19F95EeVfBt8GNXeXwSpeEb60it+eexw2dKB3Ep0cYxgFkRJP7vMpb/JpyBqvF/dALUJMXwKnqBTA1qPQvS9ZXoBeg7kbFOuhVQ+U/WdbN4CiiJniVewHK0j4vpy710UKAxF65/d+WuYBQ2jXBq9z+b8v8t8uN2SZLvFDbkOCrtv7baP1D4qva+m/LTD0h7ls2oD6qGxZ95ZZ+trMTqhsE33bF1htK3U5Z1/1AYxQ3MPhV52/Cmdw44woQX0Or2HiDiVsWdhXixpA7OPZN7uIqB3/fXVxR21XwrbqTLJi2y4bl9/MwQHlDw2+pdqk5aHVD4usodqllad9FeYsHENK+FO9Q2LUDr9qdZp/8DMlq60LVN5du53IEr86ly8A/kLl0drGZR3oA0MOoSo/2QdHDqW1TC1n3gok4m+C9IRHHzpqbddSm6nwcCyoTxyk6f3CH3lf2tNituePJtekUh5e3rl/cvQZhrxP2JnfnlcNe9PZfkSkCDw18kxvzyoEvRvG+JJvLIO6AuOu6amuyc0S702TmwtpC+LFlINRiLkh3vtnfTrCrmpFO7asJv48XsujAHgn8r8/2QJuhJpuhXdVkWDkv62dNWdABk/frdFuuxHlEBgWGIwDhVZ283ymbAdjnI4Xl6vMYbIRDX3n6vmyDYlR3bfgqz9+X7Tee6pvGKG5g8JWbiLKQAYq7LnxV5+93ZDmey0yCOCTCZEV1A6KvPH+/W/T0o7rrw1d1/v5q7CCRtxsTVDco+soT+HUNl84DBFh5Br+e/YDi2jtz6nu8dj/PpwPhO0WdA9JAeTK/ruEKepAAq87m17WyJfSufDdGjTdAAeUZ/d1iU39ii5nrej6kZTjFSm9yNfPuye9MUaxz1VtTdE9+awpdz2+7qXqjhO7J701RqHNHeZ23CzV88Kk3rePL1M9WJ301A6dzWJsidYsxtVOTpGXlup6u4i16usVQx6nVuZ1f8lMHq3N+GDIWb5z7GLqzyS3ziLji/w==</diagram></mxfile>
```

Comment:
Remeber to udpate this file after resolving other comments from this review.
Also try to keep consistent styling in this file - you missed spaces there and there. Also one of properites does not have `+` in front of it.

The plus sign is a symbol representing data access modifier in uml.
- `+` means `public`
- `-` means `private`
So it is really important : )

Reply:
(and delete this field)